scrapers/opic_scraper.py

```diff
@@ -0,0 +1,71 @@
+import pandas as pd
+import numpy as np
+import time
+from selenium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+
+def scrape_opic(search_term):
+    return OpicScraper().scrape(search_term)
+
+class OpicScraper(object):
+
+    def __init__(self):
+        pass
+
+    def scrape(self, search_term):
+
+        projectName = None
+        projectURL = None
+        projectStatus = None
+        NameList = []
+        urlList = []
+
+        browser = webdriver.Chrome("/usr/local/bin/chromedriver")
```

Comment:
This path is fixed here, but chromedriver won't necessarily be in this location on all machines. Can you remove the argument entirely? i.e.: `browser = webdriver.Chrome()`. I can confirm that the script works this way otherwise.